### PR TITLE
6503 bingx trade new order error

### PR DESCRIPTION
**Pull request by alysonturing**
### <samp>🤖 Generated by Copilot at 5c67799</samp>

Added four new components to the repository and improved the BingX Trade New Order component to handle price parameters better. Updated the `bingx` package version and the `pnpm-lock.yaml` file accordingly.

### <samp>🤖 Generated by Copilot at 5c67799</samp>

> _We're updating our packages, we're fixing our code_
> _We're adding new components to lighten our load_
> _Heave away, me hearties, on the count of three_
> _We're sailing with `BingX` and the latest `pnpm`_

### <samp>🤖 Generated by Copilot at 5c67799</samp>

*  Bump up the version of the BingX package and the BingX Trade New Order component to 0.1.2 ([link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-e3ce12deae82dd360db064c144edcb44b0459f2bc6b57de7e0f4cfa5b35a0937L5-R5), [link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-37e6e279bc7d94ca0ed853df4c0467bb6a7c0cfcb8f17e156323f8f896cd1ae5L3-R3))
*  Simplify and fix the parsing of the takerProfitPrice and stopLossPrice parameters for the BingX Trade New Order component by using the bingx.convertToFloat helper function ([link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-e3ce12deae82dd360db064c144edcb44b0459f2bc6b57de7e0f4cfa5b35a0937L64-L71), [link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-e3ce12deae82dd360db064c144edcb44b0459f2bc6b57de7e0f4cfa5b35a0937L79-R72))
*  Add the Hotjar, Retool, Wati, and Wealthbox components to the `pnpm-lock.yaml` file ([link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR1926-R1928), [link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR3583-R3585), [link](https://github.com/PipedreamHQ/pipedream/pull/6525/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR4914-R4919))

### CODE REVIEW

File: components/bingx/actions/trade-new-order/trade-new-order.mjs [hunk 1]
```diff
@@ -2,7 +2,7 @@ import bingx from "../../bingx.app.mjs";
 
 export default {
   name: "BingX Trade New Order",
-  version: "0.1.1",
+  version: "0.1.2",
```
Comment (dylburger) [inline components/bingx/actions/trade-new-order/trade-new-order.mjs:5]:
```suggestion
  version: "0.1.2",
```

Comment (dylburger) [reply components/bingx/actions/trade-new-order/trade-new-order.mjs:5]: Removed unnecessary double quotes

File: components/bingx/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/bingx",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (dylburger) [inline components/bingx/package.json:3]:
```suggestion
  "version": "0.1.2",
```

Comment (dylburger) [reply components/bingx/package.json:3]: Removed unnecessary double quotes

**Review by luancazarine — APPROVED** [review]
Hi @alysonturing, thanks for your contribution! LGTM!

**Comment by vunguyenhung** [comment]
Hi, I couldn't test this as I don't have deposit on BingX. Hence, I will make this ready for release.

**Pull request merged**